tests/usbus/Makefile

```
@@ -0,0 +1,22 @@
+include ../Makefile.tests_common
+USEMODULE += embunit
+USEMODULE += usbus
+FEATURES_PROVIDED += periph_usbdev
+
+# USB device vendor and product ID
+# pid.codes test VID/PID, not globally unique
+USB_VID ?= 1209
+USB_PID ?= 0001
+
+CFLAGS += -DUSB_CONFIG_VID=0x$(USB_VID) -DUSB_CONFIG_PID=0x$(USB_PID)
+
+include $(RIOTBASE)/Makefile.include
+
+.PHONY: usb_id_check
+usb_id_check:
+	@if [ $(USB_VID) = $(DEFAULT_VID) -o $(USB_PID) = $(DEFAULT_PID) ] ; then \
```

`-o` is a bash-ism (and Murdock complains about it.
```suggestion
	@if [ $(USB_VID) = $(DEFAULT_VID) ] || [ $(USB_PID) = $(DEFAULT_PID) ] ; then \
```

---

See also [POSIX recommendations](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/test.html#tag_20_128_16)

---

[Sorry](https://github.com/RIOT-OS/RIOT/pull/12146)

---

Maybe just replace [this line](https://github.com/RIOT-OS/RIOT/blob/master/.murdock#L1) with `#!/usr/bin/env bash` ?
I'm wondering why `/bin/sh` is still used instead of bash and what's wrong with bashing.

---

> […] and what's wrong with ~bashing~ (oups ;) ) bash-ism.

~~One dependency less~~ (ok since we use already plenty of them this argument is invalid) and assuming that bash is the default shell (which it is often, but not always ;-)).

---

I don't mind changing this as I prefer to keep things POSIX compatible where possible

---

Changed to the POSIX alternative